### PR TITLE
Subdomain multicluster

### PR DESCRIPTION
**Pull request by sivamu**
# Type of PR

- [x] Documentation changes
- [x] Code changes
- [ ] Test changes
- [ ] CI-CD changes
- [ ] GitHub Template changes

## PR Checklist

- [x] I have updated the documentation accordingly.
- [x] I have added tests to cover my changes.
- [x] All new and existing tests passed.
- [x] My code follows the code style of this project.
- [ ] I ran lint checks locally prior to submission.
- [ ] Have you checked to ensure there aren't other open Pull Requests for the same update/change?

## Purpose of PR
Modifying app endpoints to use subdomains
Updating docs, setup steps
Rebased with multi-cluster changes in #34

## Does this introduce a breaking change

- [x] YES
- [ ] NO

## Validation

- [ ] Unit tests updated and ran successfully
- [x] Update documentation or issue referenced above

Rather than using a single endpoint for the whole cluster, each app will have its own endpoint. Updated the docs, deployment templates to reflect the new domain format.

## Issues Closed or Referenced

- Closes https://github.com/retaildevcrews/wcnp/issues/547

### CODE REVIEW

**Comment by sivamu** [comment]
Will merge the PR after #34 gets merged into main.

**Review by jofultz — APPROVED** [review]

**Review by joaquinrz — APPROVED** [review]

**Pull request merged**
